webrender/src/renderer.rs

```diff
@@ -20,6 +20,7 @@ use internal_types::{TextureUpdateDetails, TextureUpdateList, PackedVertex, Rend
 use internal_types::{ORTHO_NEAR_PLANE, ORTHO_FAR_PLANE, DevicePoint};
 use internal_types::{SourceTexture, BatchTextures, TextureSampler, GLContextHandleWrapper};
 use ipc_channel::ipc;
+use prim_store::{ImagePrimitiveGpu};
```

Comment:
Yeah, this is a red flag for me. Renderer shouldn't need to know anything about the primitive store, at least directly.